src/test/java/uk/ac/stfc/facilities/mailing/mailchimp/MailchimpInternalHttpClientTest.java

```diff
@@ -69,7 +69,7 @@ public int hashCode() {
 
     // dash is required in the api key
     private static final MailchimpClientConfiguration EXAMPLE_CONFIGURATION =
-            new MailchimpClientConfiguration("abcd-ab1");
+            new MailchimpClientConfiguration("abcd-ab1","devproxy-api.facilities.rl.ac.uk/mailchimp");
```

Comment:
It looks like the  MailchimpClientConfiguration class no longer adds a `/3.0/` part on the end, and it doesn't look like the proxy adds it. Should this string have it on the end?

Reply:
Added /3.0/ at end